wpt/package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "bundle": "rollup -c rollup.config.js runner.js",
     "javy": "../target/release/javy compile -o bundle.wasm bundle.js",
-    "wasmtime": "wasmtime bundle.wasm",
+    "wasmtime": "wasmtime --env JAVY_WPT=1 bundle.wasm",
```

Comment:
I don't know much about how the `wpt `stuff is built and ran yet but could we run it with the `experimental_event_loop` feature turned on? Would that help avoid needing to pass and checking this `JAVY_WPT`?